cypress/integration/options/axes.cy.js

```diff
@@ -152,3 +169,114 @@ describe('Options - Horizontal axis', () => {
         })
     })
 })
+
+describe('Options - Auto-generated axis title', () => {
+    it('navigates to the start page and add a data item', () => {
+        goToStartPage()
+        openDimension(DIMENSION_ID_DATA)
+        selectDataElements([TEST_DATA_ELEMENT_NAME])
+        clickDimensionModalUpdateButton()
+        expectVisualizationToBeVisible(VIS_TYPE_COLUMN)
+    })
+    describe('Single item - single axis', () => {
+        it('opens Options -> Axes', () => {
```

Comment:
There is some inconsistency of the phrasing of the test names in the test files: e.g. open vs opens, set vs sets. See line 182 vs 186, 190.

Not a big deal, but it would read better if it were consistent.

Reply:
Didn't see this comment until after I merged it (got trolled by GH... again). But I'll keep this in mind for sure!